### ISSUE TITLE
Github Desktop not installing on Windows 11 Pro

### ISSUE THREAD

**Issue by Yuliana0922** [desktop/desktop#19076]
### The problem

Can't install Github Desktop on my computer. Downloaded from official source.

Things I've tried:

- Restarting my computer.
- Running the installer as an administrator.
- Running the Windows .NET repair tool (NetFxRepairTool.exe).
- Running sfc /scannow.
- Deleting the SquirrelTemp and Temp folders.

No antivirus software is running.

Log is uploaded below.

Help! 
Thanks.

### Release version

N/A

### Operating system

Windows 11 Pro

### Log files

[SquirrelSetup.log](https://github.com/user-attachments/files/16553078/SquirrelSetup.log)

### Screenshots

![Screenshot 2024-08-08 152254](https://github.com/user-attachments/assets/1c21b9a5-af54-4424-8755-2f64c0108438)

**Comment by Yuliana0922** [comment]
Update: Tried the MSI. Ran both as administrator and not.

It briefly flashes this before closing immediately. 
![Screenshot 2024-08-08 153722](https://github.com/user-attachments/assets/62db911f-dca2-49b1-9f92-23eb63dd5e97)

**Comment by steveward** [comment]
Thanks for the report. Unfortunately I'm not seeing anything helpful in the log file related to GitHub Desktop, since it looks like the last entries in the log file are from 2023. Can you attempt the install a few times to see if anything gets added to the log file? Can you open Event Viewer and see if there is anything there related to GitHub Desktop when you attempt to install?

**Comment by Yuliana0922** [comment]
Hi! I have an additional issue. Now, every time I try to install, it throws the same error, but it doesn't open the log when I click "Open Setup Log". When I click it, nothing happens at all. I've exported the error log, but I couldn't find anything even remotely related to Git or Github,  but maybe I don't know where to look. I've pasted it below:

https://drive.google.com/file/d/15cXMDWU0HRP-6oLFSGVtItmSqxSn_nLf/view?usp=sharing


Sorry for being such a hassle and thanks for your help.
